test-common/src/main/java/com/mentoring/database/DbUpdates.java

```diff
@@ -22,6 +25,15 @@ public static void setLastAccessValueByEmailAddress(Database connection, String
         waitForCacheUpdate();
     }
 
+/*    @Step("Updating production queue end time to be finished in the database.")
+    public static void setEndTimeByFactoryId(Database connection, String factoryId) {
+        connection.runUpdateLongColumnValue(
```

Comment:
Please check runUpdateLongColumnValue(String queryString, Consumer<PreparedStatement> stringMapper, Consumer<PreparedStatement> intMapper, TableColumn resultColumn) in Database.java and remove the last parameter, it is unused (TableColumn resultColumn).

Reply:
Also please refactor setLastAccessValueByEmailAddress method accordingly (TableColumn.ACCESS_TOKEN__LAST_ACCESS is not needed)

Reply:
Fixed, thx.